### DISCUSSION TITLE
how can I use `or` condition to select something with ent

### DISCUSSION THREAD

**Discussion by dengzuxuan** [ent/ent#2704] [Q&A]
just as title describe , I have two query condition , and I need use it together

**Comment by a8m** [comment]
Hey @dengzuxuan 👋 

Did you see [this example](https://entgo.io/docs/predicates#disjunction-or) on the website?

```go
client.Pet.
    Query().
    Where(
        pet.Or(
            pet.HasOwner(),
            pet.Not(pet.HasFriends()),
        ),
    ).
    All(ctx)
```

**Comment by dengzuxuan** [comment]
thank u very much!



在 2022-06-30 17:00:58，"Ariel Mashraki" ***@***.***> 写道：

Hey @dengzuxuan👋

Did you see this example on the website?

client.Pet.

    Query().

    Where(

        pet.Or(

            pet.HasOwner(),

            pet.Not(pet.HasFriends()),

        ),

    ).

    All(ctx)
